big-int/src/operators/less_than.cpp

```diff
@@ -32,5 +32,40 @@
 
 namespace libbig
 {
+bool largeInt::operator<(const largeInt &a)
+{
+    if(this->sign == NEGATIVE && a.sign == POSITIVE)
+    {
+        return 1;
+    }
+    if(a.sign == NEGATIVE && this->sign == POSITIVE)
+    {
+        return 0;
+    }
+    if(this->number.length() > a.number.length())
+    {
+        return 1^this->sign;
+    }
+    if(this->number.length() < a.number.length())
+    {
+        return 0^this->sign;
+    }
 
+    for(int i=0;i<a.number.length();i++)
+    {
+        if(this->number[i]==a.number[i])    
```

Comment:
this one is also redundant.

Reply:
It is not redundant as if I remove this I need to add another condition to check if `this->number[i]>a.number[i]`

Reply:
```C++
if (this->number[i] < a.number[i]) {
    return !(1 ^ this->sign);
}
if (this->number[i] > a.number[i]) {
    return !(0 ^ this->sign);
}
```
i think these two conditions are sufficient, isn't?

Reply:
Yes these are sufficient but it won't make much difference to the efficiency of the program. If you insist I'll make the changes after some time.

Reply:
okay no problem